examples/common/python/Makefile

```diff
@@ -34,10 +35,11 @@ build :
 	cd $@ 
 
 install:
-	@echo INSTALLING EGG FILE =================
-	easy_install $(EGG_FILE)
+	@echo INSTALLING WHEEL FILE =================
+	pip3 install $(WHEEL_FILE)
 
 clean:
+	pip3 uninstall --yes $(WHEEL_FILE)
 	rm -rf build deps dist *.egg-info
```

Comment:
I guess we can remove this line since egg is not used ?

Reply:
All these folders are still created and egg-info is created by pip. pip identifies whether a package is installed or not based on the existance of "egg-info" folder. It has nothing to do with egg packaging.

Reply:
ok. I was reading in below link which mentions -> Wheel uses PEP376-compliant .dist-info directories. Egg used .egg-info.
https://packaging.python.org/discussions/wheel-vs-egg/